Gemfile

```
@@ -2,4 +2,22 @@
 
 source 'https://rubygems.org'
 
+gem 'git-version-bump', '~> 0.15' if RUBY_VERSION >= '2.3.0'
```

```suggestion
gem 'git-version-bump', '~> 0.15'
```

Not needed since Ruby 2.2 support was dropped, no?